contracts/priv/cloud-priv.yml

```diff
@@ -619,6 +736,66 @@ components:
                     user:
                       readOnly: true
                       $ref: '#/components/schemas/Link'
+    OnboardingWithIDsRequest:
+      type: object
+      properties:
+        user:
+          $ref: '#/components/schemas/Identity'
+        org:
+          $ref: '#/components/schemas/Identity'
+        config:
+          type: object
+          properties:
+            bucket:
+              type: string
+              description: name of non system bucket
+            retentionPeriodSeconds:
+              type: integer
+              description: retention period of non system bucket
+            limit:
+              $ref: '#/components/schemas/Limit'
+        returnToken:
+          type: bool
+          description: if true will return the auth token in the response
+      required:
+        - username
+        - org
+        - config
+    OnboardingUserRequest:
+      type: object
+      properties:
+        user:
+          $ref: '#/components/schemas/Identity'
+        orgID:
+          type: string
+          description: organization id to add user to
+        role:
+          type: string
+          description: Role of the User
+          enum:
+            - owner
+            - member
+      required:
+        - user
+        - orgID
+        - Role
+    OnboardingWithIDsResponse:
+      type: object
+      properties:
+        user:
+          $ref: '#/components/schemas/Identity'
+        org:
+          $ref: '#/components/schemas/IdentityLinks'
+    OnboardingWithTokenResponse:
+      type: object
+      properties:
+        user:
+          $ref: '#/components/schemas/Identity'
+        org:
+          $ref: '#/components/schemas/Identity'
+        auth:
```

Comment:
Similar to comment above; should this be named `token` instead? If this is really an `auth token`, then this name is fine.

Reply:
FYI , anything under contracts is generated from files under /src. So you don't have to repeat the comments if you don't want to :-)

Reply:
What do you want to call it

Reply:
Let's go with `token`; that's what it's called in the authorizations endpoints.